.bazelrc

```diff
@@ -2,5 +2,8 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 
-#https://docs.opentitan.org/doc/rm/c_cpp_coding_style/#cxx-version specifies
+# https://docs.opentitan.org/doc/rm/c_cpp_coding_style/#cxx-version specifies
 build --cxxopt='-std=c++14'
```

Comment:
Should there be one for `c11` too?

Reply:
Oh, I guess I missed that in the doc. I'll fix that.